frontend/src/components/organisms/header.tsx

```diff
@@ -40,6 +41,7 @@ const Header = memo(() => {
         }}
       >
         <ChatProfiles />
+        <SelectedAssistantDisplay />
```

Comment:
can chat profiles co exist with assistants?

Reply:
they should not